core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -161,6 +163,8 @@ public Object getFieldValue(MasterWorkerInfo o) {
   /** Keeps track of workers which are no longer in communication with the master. */
   private final IndexedSet<MasterWorkerInfo> mLostWorkers =
       new IndexedSet<>(ID_INDEX, ADDRESS_INDEX);
+  /** Stores the cluster metrics. */
+  private final MetricsStore mMetricsStore;
```

Comment:
does this need to be here, or can we move it to `AbstractMaster` like the other fields from `MasterContext`?